### PR TITLE
Added Largest Series Product exercise

### PR DESCRIPTION
**Pull request by icyrockcom**
Let me know how it looks.

### CODE REVIEW

File: exercises/largest-series-product/examples/src/LargestSeriesProduct.purs [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+import Data.Traversable (sequence)
+
+spans :: forall a. Int -> Array a -> Array (Array a)
+spans n a
+  | length a < n = []
+  | otherwise    = take n a : spans n (drop 1 a)
+
+toIntArray :: String -> Maybe (Array Int)
+toIntArray = toCharArray
+             >>> map (singleton >>> fromString)
```
Comment (paf31) [inline exercises/largest-series-product/examples/src/LargestSeriesProduct.purs]: `map f >>> sequence` is just `traverse f`

**Comment by paf31** [comment]
Thanks!

**Comment by icyrockcom** [comment]
Thanks @lpil @paf31!

File: exercises/largest-series-product/examples/src/LargestSeriesProduct.purs (hunk 1 above)
Comment (icyrockcom) [inline exercises/largest-series-product/examples/src/LargestSeriesProduct.purs]: Thanks @paf31, missed that one, I'll try to remember :) I've pushed an update https://github.com/icyrockcom/xpurescript/commit/38bcfc6e314141fa96e0c2c97b517ea9c002c628 if you prefer to merge that one.

**Pull request merged**